apriltag_ros/src/common_functions.cpp

```diff
@@ -310,6 +310,15 @@ AprilTagDetectionArray TagDetector::detectTags (
     tag_detection.pose = tag_pose;
     tag_detection.id.push_back(detection->id);
     tag_detection.size.push_back(tag_size);
+    //inclusion of tag corners' pixel coordinates
+    tag_detection.pix_tl[0]=detection->p[3][0];
+    tag_detection.pix_tl[1]=detection->p[3][1];
+    tag_detection.pix_tr[0]=detection->p[2][0];
+    tag_detection.pix_tr[1]=detection->p[2][1];
+    tag_detection.pix_br[0]=detection->p[1][0];
+    tag_detection.pix_br[1]=detection->p[1][1];
+    tag_detection.pix_bl[0]=detection->p[0][0];
+    tag_detection.pix_bl[1]=detection->p[0][1];
```

Comment:
Assuming the message has an array of pixels, then this will flatten into a for loop